src/components/NavigationDrawer.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { StyleSheet, View, Image, TouchableOpacity } from 'react-native'
+import BottomSheet from 'reanimated-bottom-sheet'
+import Icon from 'react-native-vector-icons/FontAwesome5'
+
+import logo from '../../assets/logo.png'
+
+type NavigationBarProps = {
+    navigation: StackNavigationProp<>;
+  }
+
+const styles = StyleSheet.create( {
+  navigationPanel: {
+    height: '100%',
+    backgroundColor: '#f9f9fa',
+    opacity: 0.92,
+  },
+  navigationPanelHeader: {
+    backgroundColor: '#f9f9fa',
+    paddingTop: 7,
+    borderTopLeftRadius: 20,
+    borderTopRightRadius: 20,
+    alignItems: 'center',
+    opacity: 0.92,
+  },
+  navigationPanelHandle: {
+    width: 80,
+    height: 2,
+    borderRadius: 5,
+    backgroundColor: '#00000040',
+  },
+  navigationPanelIconContainer: {
+    flexDirection: 'row',
+    paddingTop: '5%',
+    paddingBottom: '10%',
+  },
+  navigationIconSpacing: {
+    paddingRight: 30,
+    paddingLeft: 30,
+  },
+  navigationPanelButtonContainer: {
+    flexDirection: 'row',
+    justifyContent: 'space-around',
+  },
+} )
+
+const Buttons = [
+  [ 'clock', 'WIP' ],
+  [ 'bookmark', 'WIP' ],
+  [ 'search', 'Search' ],
+  [ 'square', 'WIP' ],
+  [ 'ellipsis-h', 'WIP' ],
+]
+
+const NavigationBar: React.FC<NavigationBarProps> = ( { navigation } ) => {
+  const renderNavigationHeader = () => (
+    <View style={styles.navigationPanelHeader}>
```

Comment:
we should be using `native-base` for the layout components, not the `react-native` components

Reply:
https://docs.nativebase.io/docs/CheatSheet.html so according to this `Container` is alternative for `View` (that cheatsheet is outdated)  anyway when I try using that then it just pushes stuff out. 
With `View` I can design it how I want it to look because `Container` brings stuff along that will require modifying that component and it is just easier to use `View`.

Reply:
Container is actually the main body of a screen component, so it wouldn’t make sense either. I don’t really see what else would be appropriate here, I may be wrong, but I think View is fine too?

Reply:
native-base also has components for all the react components, including view. All I am suggesting is that we change the references across from react-native to native-base